actions/update-charts-pr/action.yaml

```
@@ -0,0 +1,96 @@
+name: "Update site-values and create PR"
```

```suggestion
name: "Update Chart Version in Target Repository"
```

Apply also in all references where it says "site-values" instead of target repo